package.json

```
@@ -56,11 +56,11 @@
 		"lucide-react": "^0.546.0",
 		"lz-string": "^1.5.0",
 		"monaco-editor": "^0.52.2",
-		"next": "^15.5.6",
+		"next": "^15.5.9",
 		"next-themes": "^0.4.6",
 		"postgres": "^3.4.7",
-		"react": "^19.2.0",
-		"react-dom": "^19.2.0",
+		"react": "^19.2.3",
```

Same, let's use strict version 
```suggestion
		"react": "19.2.3",
```